packages/merge-on-green/src/merge-logic.ts

```diff
@@ -669,6 +692,17 @@ export async function mergeOnGreen(
       await mergeOnGreen.merge(owner, repo, pr, prInfo, github);
       merged = true;
     } catch (err) {
+      if (err.message.includes('not authorized to push to this branch')) {
```

Comment:
Is this error message from the API? Is there a 403 response code or something we can use? I'm concerned this string might change.